pgmpy/models/DynamicBayesianNetwork.py

```diff
@@ -440,6 +440,53 @@ def remove_cpds(self, *cpds):
                 cpd = self.get_cpds(cpd)
             self.cpds.remove(cpd)
 
+    def get_cardinality(self, node=None):
+        """
+        Returns the cardinality of the node.If node=None returns all the cardinalities.
+
+        Parameter
```

Comment:
@lohani2280 Use `Parameters`, the doc generator parses that only I think.

Reply:
@ankurankan Yeah sure, i'll update it.